mod/videos.php

```diff
@@ -114,7 +111,7 @@ function videos_content(App $a)
 	// videos/name/video/xxxxx/edit
 
 
-	if ((Config::get('system', 'block_public')) && (!local_user()) && (!remote_user())) {
+	if (Config::get('system', 'block_public') && !Session::isAuthenticated()) {
 		notice(L10n::t('Public access denied.') . EOL);
```

Comment:
This should throw a `ForbiddenException` like this:
```php
throw new HTTPException\ForbiddenException(L10n::t('Public access denied.'));
```

Reply:
When we decide to do it this way, we have to do it at all places.

Reply:
Fair enough.

Reply:
Fair enough.

Reply:
If you had said: "Do it so", then I would had changed it at every place :-)

Reply:
But it wouldn't have been in the scope of this already large PR.